src/bio/smd/tests/smd_ut.c

```diff
@@ -251,6 +251,15 @@ static const struct CMUnitTest smd_uts[] = {
 
 int main(int argc, char **argv)
 {
+	int	rc;
+
+	rc = ABT_init(0, NULL);
```

Comment:
Shouldn't there be an ABT_finalize() call?

Reply:
indeed it should be there, I will update this patch, once I figure out why run_tests.sh is failing in CI.